src/GraphODataTemplateWriter/Extensions/OdcmModelExtensions.cs

```diff
@@ -42,7 +42,7 @@ private static OdcmNamespace GetOdcmNamespace(OdcmModel model)
         public static IEnumerable<OdcmClass> GetComplexTypes(this OdcmModel model)
         {
             var @namespace = GetOdcmNamespace(model);
-            return @namespace.Classes.Where(x => x is OdcmComplexClass);
+            return @namespace.Classes.Where(x => x is OdcmComplexClass && x.CanonicalName() != "microsoft.graph.Json");
```

Comment:
Not relevant to your changes but why is this not microsoft.graph.json to keep consistent with our casing pattern? Should probably push back on that.